testutil/channel_wait.go

```diff
@@ -0,0 +1,35 @@
+package testutil
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+func ChannelWaitForValue(t *testing.T, waitOn interface{}) interface{} {
```

Comment:
can we have here options interface allowing caller to set custom timeout value?

Reply:
Sure, I will have two functions in this file. One with a default value set and the other where the caller can set how long to wait.

Reply:
you can do one function with options pattern like [here](https://github.com/VolantMQ/volantmq/blob/master/connection/writerOptions.go)

Reply:
I don't really see an advantage to doing that for what is a single option. It's just going to create a complex way of doing an obvious thing.